versions/draft-3/SPEC.md

```diff
@@ -1020,8 +1020,7 @@ task memory_test {
 $parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
```

Comment:
My own $0.02 here is that I'd prefer just to remove these grammar blocks as we make modifications. IMO hermes grammer isn't ubiquitous enough for those to really be helpful

Reply:
@geoffjentry I think I understand the hermes concern, but I am not sure about the grammar block removal. Do you mean, remove the parameter_meta section? merge it with the meta section? 

If both will be removed, I don't know where to place the extra information we need.

Reply:
I'm specifically talking about lines 1019 to 1025. I'd wait and see if anyone else agrees prior to doing something

Reply:
I am fine with this, if we merge the `parameter_meta` with the `meta` section, and remove the restriction on keys. Currently, a `parameter_meta` key has to be an input parameter.

Reply:
I’m not commenting on the content of your change at all, except that you’re enhancing a think should disappear.

All I’m saying is that I’d like to remove the Hermès grammar blocks out of The spec as we find ourselves updating things.

I agree with your actual suggestion

Reply:
Oh, now I get it. So, what is a grammar example I should follow?

Reply:
It has admittedly been as hoc but I feel like the current replcement is better examples.

Ultimately I’d like to see an appendix with BNF but right now I feel like the Hermès stuff just adds visual noise with little value

Reply:
But like I aid don’t do it unless there are other folks agreeing